src/drive/web/modules/selection/SelectionBar.jsx

```diff
@@ -19,8 +19,22 @@ const mapDispatchToProps = dispatch => ({
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(({ selectionModeActive, ...rest }) => (
-  <div style={{ display: selectionModeActive ? 'inherit' : 'none' }}>
-    <SelectionBar {...rest} />
-  </div>
-))
+)(({ selectionModeActive, actions, ...rest }) => {
+  const convertedActions = DriveActionToSelectionBarAction(actions)
+  return (
+    <div style={{ display: selectionModeActive ? 'inherit' : 'none' }}>
+      <SelectionBar actions={convertedActions} {...rest} />
+    </div>
+  )
+})
+
+const DriveActionToSelectionBarAction = driveActions => {
+  let actions = []
+  driveActions.map(driveAction => {
+    const actionName = Object.keys(driveAction)[0]
```

Comment:
I find it weird to have to take the first key for the actionName. For hr the first key is "icon", it does not correspond to the `actionName` variable here.

Reply:
Should the action have a property `displayInSelectionBar` that we would set to false for hr ? I think it'd be a bit cleaner since the selectionBar should not have to care about `hr`.